src/redux/playlistsSlice.js

```diff
@@ -13,16 +13,23 @@ export const playlistsSlice = createSlice({
         "before long operation in load reducer",
         new Date().getTime() / 1000
       );
-      for (let index = 0; index < 2000000000; index++) {}
+      // for heavy tasks pls create async methods and use redux thunk
+      // for (let index = 0; index < 2000000000; index++) { }
       console.log(
         "after long operation in load reducer",
         new Date().getTime() / 1000
       );
-      state.list = data.list;
+      return {
+        ...state,
+        list: data.list
+      }
```

Comment:
state is immutable dont chande it, instead return new object with old state and updated fields